.gitignore

```diff
@@ -15,3 +15,7 @@
 /log/*
 !/log/.keep
 /tmp
+
+# ignore precompiled assets
+public/assets/
+*.DS_Store
```

Comment:
you shouldn't need these rules - the `*.DS_Store` should sit in your global gitignore